packages/components/ContextualMenu/src/Submenu.tsx

```diff
@@ -70,7 +70,9 @@ export const Submenu = compose<SubmenuType>({
         setInitialFocus: shouldFocusOnMount,
       },
       container: Platform.select({
-        macos: {},
+        macos: {
+          ...useKeyDownProps(onDismiss, 'ArrowLeft'),
```

Comment:
Note: This should flip in RTL. I'm thinking I'll do that in a separate change.

Reply:
@ppatboyd Would this be a good to have on Win32? Having `ArrowLeft` close a submenu?

Reply:
Tested. This won't work on win32 because the native behavior of arrow keys is to move focus. In order to supress that and _actually_ override the keypress, we'd need to specify the prop `keyDownEvents=[{code: 'ArrowLeft'}]`. That would be a good thing to add to `useKeyProps / useKeyUpProps / useKeyDownProps`....